back/app/models/project.rb

```diff
@@ -133,6 +133,13 @@ class Project < ApplicationRecord
     where(id: project_ids)
   }
 
+  class << self
+    def search_ids_by_all_including_patches(term)
+      result = defined?(super) ? super : []
```

Comment:
I tried to make this code self-documented, hence the name has `_including_patches`. But still, probably, it's not so easy to read. What part is not clear yet?